appProject/appProject/SearchViewController.swift

```diff
@@ -0,0 +1,213 @@
+//
+//  SearchViewController.swift
+//  appProject
+//
+//  Created by Валентин Коскин on 06.10.2022.
+//
+
+import UIKit
+/// экран поиска
+class SearchViewController: UIViewController, UISearchResultsUpdating {
```

Comment:
```suggestion
class SearchViewController: UIViewController {
```
UISearchResultsUpdating   extension